python-test/features/policies.feature

```diff
@@ -0,0 +1,8 @@
+@policies
+Feature: policy creation
+
+  Scenario: Create a policy
+    Given the Orb user logs in
+      And that an agent already exists and is online
+    When a new policy is created
+    Then referred policy must be listened on the orb policies list
```

Comment:
Should we also check in the orb agent if the creation of this policy had effect?

Reply:
we will only be able to see the effects in the orb agent after creating the dataset